Tests/UnitTests/Fatras/Digitization/ChannelMergerTests.cpp

```diff
@@ -0,0 +1,96 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "ActsFatras/Digitization/ChannelMerger.hpp"
+#include "ActsFatras/Digitization/DigitizationError.hpp"
+
+#include <Acts/Tests/CommonHelpers/FloatComparisons.hpp>
+#include <Acts/Utilities/Definitions.hpp>
+#include <Acts/Utilities/Helpers.hpp>
+#include <Acts/Utilities/ParameterDefinitions.hpp>
+
+namespace ActsFatras {
+
+BOOST_AUTO_TEST_SUITE(Digitization)
+
+BOOST_AUTO_TEST_CASE(ChannelMerger1D) {
+  ChannelMerger cm;
+
+  // Some cells
+  Cell cell0(5, 10.5);
+  Cell cell1(6, 11.5);
+  Cell cell2(7, 12.5);
+
+  using Channel1D = Channel<double, Acts::eBoundLoc0>;
+
+  // Digital clustering test
+  std::vector<Channel1D> channels = {{{cell0}, 1., {5}},
+                                     {{cell1}, 1., {5}},
+                                     {{cell2}, 1., {5}},
+                                     {{cell0}, 0.5, {6}}};
+
+  BOOST_CHECK_EQUAL(channels.size(), 4u);
+  auto mergedChannels = cm.merge<double, Acts::eBoundLoc0>(channels);
```

Comment:
```suggestion
  auto mergedChannels = cm.merge(channels);
```
The template parameters should auto-deductible.

Reply:
Removed.